### ISSUE TITLE
Wrong position for non prefixed hook name on new line

### ISSUE THREAD

**Issue by grappler** [WordPress-Coding-Standards/WordPress-Coding-Standards#1495]
Atom PHPCS was failing due to "Invalid point encountered in the attached message". I opened an issue 
 https://github.com/AtomLinter/linter-phpcs/issues/306

Though after looking into it again, it seems to be an issue with the Sniff `WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound`

The code that I have is as follows:

```php
return apply_filters(
	'content-types-post-types',
	[
		PostType\Post::NAME             => PostType\Post::class,
	]
);
```

The error shows OK if I have the code as follows:

```php
return apply_filters( 'content-types-post-types',
	[
		PostType\Post::NAME             => PostType\Post::class,
	]
);
```

The command line works regardless but the line number is incorrect.

Most likely `$first_non_empty` should be used instead of `$parameters[1]['start']` in 

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/784fefe0183d11d0bd7609e4701beb9c9782f956/WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php#L747
